script.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable no-plusplus */
+const navlistmobile = document.querySelector('.nav-list-mobile');
+const wrap = document.querySelector('#wrap');
+const menubarfluid = document.querySelector('.menu-bar-fluid');
+const featuresection = document.querySelector('#feature-section');
+
+wrap.addEventListener('click', () => {
+  navlistmobile.classList.toggle('nav-list-mobile-change');
+  menubarfluid.classList.toggle('menu-bar-fluid-background-change');
+  wrap.classList.toggle('change-wrap');
+});
+
+const coursesiconmobile1 = document.querySelector('#coursesiconmobile');
+const joiniconmobile1 = document.querySelector('#joiniconmobile');
+const sponsericonmobile1 = document.querySelector('#sponsericonmobile');
+const latesticonmobile1 = document.querySelector('#latesticonmobile');
+
+const menuArray = [coursesiconmobile1, joiniconmobile1, sponsericonmobile1, latesticonmobile1];
+for (let i = 0; i < menuArray.length; i++) {
+  menuArray[i].addEventListener('click', () => {
+    menubarfluid.classList.remove('menu-bar-fluid-background-change');
+    navlistmobile.classList.remove('nav-list-mobile-change');
+    wrap.classList.remove('change-wrap');
+  });
+}
+
+const lecturesData = [
+  {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` your project using the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`. Your project must be the same as [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), you must make your project `color`, `shape`, `layout`, and `fonts` as shown on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please correct them 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235015380-7b30dadc-c3fe-4458-9947-f3b802f1479d.png)